app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    cmd, old_loc, new_loc = command
```

Comment:
Make sure you check that the `mv` input command is correct.
Also before unpacking, check that the length of `command ` is equal to thre